### PR TITLE
Minor ParamTypeFromStrictTypedPropertyRector improvements

### PR DESCRIPTION
**Pull request by staabm**

### CODE REVIEW

File: rules/TypeDeclaration/Rector/Param/ParamTypeFromStrictTypedPropertyRector.php [hunk 1]
```diff
@@ -103,6 +103,10 @@ private function decorateParamWithType(ClassMethod $classMethod, Param $param):
             return null;
         }
 
+        if ($classMethod->isAbstract()) {
```
Comment (staabm) [inline rules/TypeDeclaration/Rector/Param/ParamTypeFromStrictTypedPropertyRector.php:106]: before this PR a few things above the loop were resolved even though not necessary as a method without a body is not worth introspecting

File: rules/TypeDeclaration/Rector/Param/ParamTypeFromStrictTypedPropertyRector.php [hunk 2]
```diff
@@ -111,7 +115,6 @@ private function decorateParamWithType(ClassMethod $classMethod, Param $param):
 
         $paramName = $this->getName($param);
 
-        /** @var Assign[] $assigns */
         $assigns = $this->betterNodeFinder->findInstanceOf((array) $classMethod->getStmts(), Assign::class);
```
Comment (TomasVotruba) [inline rules/TypeDeclaration/Rector/Param/ParamTypeFromStrictTypedPropertyRector.php]: This is needed for PHPStorm IDE autocomplete.

Comment (staabm) [reply rules/TypeDeclaration/Rector/Param/ParamTypeFromStrictTypedPropertyRector.php]: good catch. fixed.

**Review by samsonasik — APPROVED** [review]
Looks good, thank you @staabm

**Pull request merged**